src/FFmpegWriter.cpp

```diff
@@ -503,6 +515,36 @@ void FFmpegWriter::SetOption(StreamType stream, std::string name, std::string va
 						c->bit_rate = (int) (mbs);
 				}
 			}
+#endif
+		} else if (name == "qp") {
+			// encode quality and special settings like lossless
+			// This might be better in an extra methods as more options
+			// and way to set quality are possible
+#if (LIBAVCODEC_VERSION_MAJOR >= 58)
+				switch (c->codec_id) {
+					case AV_CODEC_ID_AV1 :
+						c->bit_rate = 0;
+						av_opt_set_int(c->priv_data, "qp", std::min(std::stoi(value),63), 0);
+						if (strstr(info.vcodec.c_str(), "svt_av1") != NULL) {
+							av_opt_set_int(c->priv_data, "qp", std::min(std::stoi(value),63), 0);
+							av_opt_set_int(c->priv_data, "preset", 6, 0);
+							av_opt_set_int(c->priv_data, "forced-idr",1,0);
+						}
+						if (strstr(info.vcodec.c_str(), "rav1e") != NULL) {
+							av_opt_set_int(c->priv_data, "qp", std::min(std::stoi(value),255), 0);
+							av_opt_set_int(c->priv_data, "speed", 7, 0);
+							av_opt_set_int(c->priv_data, "tile-rows", 2, 0);
+							av_opt_set_int(c->priv_data, "tile-columns", 4, 0);
+							//av_opt_set(c->priv_data, "tile-row", "", 0);
+						}
+						 if (strstr(info.vcodec.c_str(), "aom") != NULL) {
+							// Hack to set tiles; libaom doesn have qp only crf
+							av_opt_set_int(c->priv_data, "crf", std::min(std::stoi(value),63), 0);
+							av_opt_set_int(c->priv_data, "tile-rows", 1, 0);		// log2 of number of rows
+							av_opt_set_int(c->priv_data, "tile-columns", 2, 0);		// log2 of number of columns
+						}
```

Comment:
"Hack to set tiles?" So if I'm understanding this right, when using the `aom` codec:
```c++
// Encode with a CRF of 15 and.... (no? default?) tiling
writer.setOption(VIDEO_STREAM, "crf", 15)
// Encode with a CRF of 15 and tile-rows=1, tile-columns=2
writer.setOption(VIDEO_STREAM, "qp", 15)
```
Is that right? Feels a bit magical for my blood. Is there any reason to do it this way, instead of just setting the `tile-*` values for `aom` in the `crf` option processing, the way it's done for `rav1e`?